pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -161,3 +185,37 @@ func (e *Executor) getMentionedAccounts(event model.NotificationEventType) ([]st
 
 	return notification.FindSlackAccounts(event), nil
 }
+
+func (e *Executor) validateApproverNum(ctx context.Context, approver string, minApproverNum int) (string, bool) {
```

Comment:
The return values are a bit tricky, so could you add a comment for them or use named return values?